simply-rets-api-helper.php

```diff
@@ -1373,6 +1394,10 @@ public static function srResidentialResultsGenerator( $response, $settings ) {
                     $compliance_markup
                   </div>
                 </div>
+                <div class="sr-screen-reader" itemprop="geo" itemscope itemtype="http://schema.org/GeoCoordinates">
+                  <meta content="$lat" itemprop="latitude">
+                  <meta content="$lng" itemprop="longitude"
```

Comment:
Awesome, thank you! Other than that I'm good to pull this in -- thanks a ton for the change! Really cool feature.

Let me know if you need any other help.